src/ClassDesignChallengeConstructors/Course.java

```diff
@@ -0,0 +1,18 @@
+package ClassDesignChallengeConstructors;
+
+public class Course {
+
+    private String courseName;
+    private String schedule;
```

Comment:
Please pay attention to property names. If we have `courseName`, `courseDescription`, we should keep this naming convention for all the properties.
On the other hand, I would not recommend to repeat `course` name for all the properties. We know we are talking about the name, schedule, and description of the Course because we are in the Course class.